### PR TITLE
fix route problem on patidex after merge

### PR DESCRIPTION
**Pull request by KevinRoig**
No comment because you see me.

### CODE REVIEW

**Review by RomainDesmidt — APPROVED** [review]
ok c'est bon !

File: src/app/pages/patidex/patidex.page.component.html [hunk 1]
```diff
@@ -1 +1 @@
-<p>patidex works!</p>
```
Comment (maxime-auneau) [inline src/app/pages/patidex/patidex.page.component.html:1]: # **The name need to change for something like (patidex-list or patidex-component)**

Comment (maxime-auneau) [reply src/app/pages/patidex/patidex.page.component.html:1]: 😾😾😾😾😾😾😾😾😾

Comment (KevinRoig) [reply src/app/pages/patidex/patidex.page.component.html:1]: [https://media.gqmagazine.fr/photos/5b991b5791715b0011253e54/16:9/w_1280,c_limit/un_emoji_pour_les______tous_3480.jpeg](url)

Comment (KevinRoig) [reply src/app/pages/patidex/patidex.page.component.html:1]: Or not.

**Review by maxime-auneau — APPROVED** [review]
### C'est insupportable

**Pull request merged**